parity/cli/mod.rs

```diff
@@ -296,7 +296,7 @@ usage! {
 
 			ARG arg_release_track: (String) = "current", or |c: &Config| c.parity.as_ref()?.release_track.clone(),
 			"--release-track=[TRACK]",
-			"Set which release track we should use for updates. TRACK can be one of: stable - Stable releases; beta - Beta releases; nightly - Nightly releases (unstable); testing - Testing releases (do not use); current - Whatever track this executable was released on.",
+			"Set which release track we should use for updates. TRACK can be one of: stable - Stable releases; nightly - Nightly releases (unstable); testing - Testing releases (do not use); current - Whatever track this executable was released on.",
```

Comment:
Hmm, we're not really doing testing releases are we? Like, what would the difference between nightly and testing be?

Reply:
We don't do testing releases no, it's not a track I've ever used. Regarding auto-releases for parity version, this is one of the trickier things; since the 2.7 onwards releases will be `stable` branch, we need a way for users on `beta` to switch to `stable` in order to continue getting updates. I feel like the most effective way of doing this is simply to communicate it clearly in a release announcement post.

Reply:
AFAIK `testing` was used for internal testing. Like units.